image/discourse_dev/Dockerfile

```diff
@@ -5,9 +5,9 @@ ENV RAILS_ENV development
 
 #LABEL maintainer="Sam Saffron \"https://twitter.com/samsaffron\""
 
-# Install for mailcatcher gem
-RUN apt update && apt install -y libsqlite3-dev \
-    && gem install mailcatcher && rm -rf /var/lib/apt/lists/*
+# Install for MailHog (https://github.com/mailhog/MailHog)
+RUN apt update && apt install -y golang-go \
+    && go get github.com/mailhog/MailHog
```

Comment:
This doesn't make mailhog available on the PATH. To run it, people would have to type

```
/root/go/bin/MailHog ...
```
(plus I'm not sure if there would be permissions problems, since the binary is build as `root`, and might be run as `discourse`)

At a minimum we should move the binary to `/usr/local/bin`. Even better would be to copy the 'multistage build' technique from [MailHog's own Dockerfile](https://github.com/mailhog/MailHog/blob/master/Dockerfile). The added bonus there is that we don't have to ship a golang development environment in our docker image.

Reply:
I would not install the full go toolset, just copy the binary to the right place

Reply:
At the moment the golang toolset is required to build the binary in the first place. If we're happy to hard-code the MailHog version, then we could fetch the pre-built `linux_amd64` binary from https://github.com/mailhog/MailHog/releases/

Reply:
I've made the required change.